client/lib/pages/add-plant.dart

```diff
@@ -40,19 +65,47 @@ class _AddPlant extends State<AddPlant> {
                 TextStyle(fontFamily: 'Libre Baskerville', color: Colors.white),
             child: Padding(
               padding: EdgeInsets.all(15.0),
-              child: ListView(
-                children: <Widget>[
-                  Form(
+              child: Form(
                     key: _formKey,
                     child: ListView(
                       children: <Widget>[
                         Text('Afbeelding'),
-                        IconButton(
-                          icon: Icon(Icons.image),
-                          onPressed: () => {},
-                          color: Colors.blue,
+                        SizedBox(height: 10),
+                        Row(
+                          children: <Widget>[
+                            Expanded(
+                              flex: 6,
+                              child: Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: <Widget>[
+                                  Image.file(
+                                    File(newPlant.imageName ?? ''),
+                                    width: 150,
+                                    height: 150,
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Expanded(
+                              flex: 4,
+                              child: Row(
+                                children: <Widget>[
+                                  IconButton(
+                                    icon: Icon(Icons.image),
+                                    onPressed: () => selectFromSource(context, ImageSource.gallery),
+                                    color: Colors.blue,
+                                  ),
+                                  IconButton(
+                                    icon: Icon(Icons.camera_alt),
+                                    onPressed: () => selectFromSource(context, ImageSource.camera),
+                                    color: Colors.blue,
```

Comment:
Idk if this is where the buttons are made for the gallery and the camera, but they are way too close to each other. Hard to know which button I am pressing.